### ISSUE TITLE
"gamepad_api_enabled": "true", MAKE STEAMVR ADD-ONS CRASH 

### ISSUE THREAD

**Issue by PantACRO4life** [psmoveservice/PSMoveService#731]
Hi, I ran into a new issue I believe or I might Have done something wrong But I cannot figure it out.
I follow all the guide from Daley Tech on YouTube, and now I reach the point where I am connected my PS move navigator with psmoveservice. Gamepad_API_Enabled is to true but when I start  everything with that set to true now make one of my ps move controller lit on and off and start steam vr under an issue which disable all the Add-Ons do to crash.  

If I unblock all Add-Ons but psmove It turn out it work so the issue point to psmove service and I dont know what to do since I cannot find allot about it.

Looking forward for your help .

**Comment by PantACRO4life** [comment]
I finally ended-up resolving the issue myself with that thread "https://github.com/HipsterSloth/PSMoveSteamVRBridge/issues/107"

I had to replace the ":" with "_" in both "psnavi_gamepad_2.json" and "psnavi_gamepad_3.json" files.
